### PR TITLE
feature/les-491 edit side bar

### PR DESCRIPTION
**Pull request by OxygenO22**

### CODE REVIEW

**Review by valer-sky — APPROVED** [review]

File: env.local [hunk 1]
```diff
@@ -0,0 +1,6 @@
+REACT_APP_BACKEND_URL = "https://lessoner-project-2w3h.onrender.com"
```
Comment (Terapool) [inline env.local:1]: This file to be added into .gitignore. Do not include it in any PR

File: src/components/NavbarStudyStudioSVGSelector.tsx [hunk 2]
```diff
@@ -15,41 +25,43 @@ type Props = {
...
+  case "network":
+    return <Network />;
+  case "instagram":
+    return <Instagram />;
+  case "facebook":
+    return <Facebook />;
+  case "linkedin":
+    return <Linkedin />;
   default:
     return <svg></svg>;
```
Comment (Terapool) [inline src/components/NavbarStudyStudioSVGSelector.tsx:66]: return null;

**Review by Terapool — CHANGES_REQUESTED** [review]

File: src/components/NavbarStudyStudioSVGSelector.tsx (hunk 2 above)
Comment (OxygenO22) [reply src/components/NavbarStudyStudioSVGSelector.tsx:66]: done

File: env.local (hunk 1 above)
Comment (OxygenO22) [reply env.local]: Done

**Review by DimitarShemet — APPROVED** [review]

**Review by Terapool — APPROVED** [review]

**Pull request merged**